<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="all_features" errors="3" failures="9" tests="31">
    <testcase classname="coverage_maxout.cpp" name="exercising tricky code paths of doctest">
      <failure message="0 != 0" type="CHECK">
        coverage_maxout.cpp(41): Expression str.compare(const_str, true) != 0 became 0 != 0
      </failure>
      <failure message="0 != 0" type="CHECK">
        coverage_maxout.cpp(42): Expression str.compare("omgomgomg", false) != 0 became 0 != 0
      </failure>
      <failure message="false" type="CHECK">
        coverage_maxout.cpp(107): Expression len_is_zero became false
      </failure>
      <failure message="Approx( 7.0 ) == 5" type="CHECK">
        coverage_maxout.cpp(113): Expression b == 5 became Approx( 7.0 ) == 5
      </failure>
      <failure message="Approx( 7.0 ) &lt;  5" type="CHECK">
        coverage_maxout.cpp(116): Expression b &lt; 5 became Approx( 7.0 ) &lt;  5
      </failure>
      <failure message="Approx( 7.0 ) &lt;= 5" type="CHECK">
        coverage_maxout.cpp(118): Expression b &lt;= 5 became Approx( 7.0 ) &lt;= 5
      </failure>
      <failure message="6 == Approx( 5.0 )" type="CHECK">
        coverage_maxout.cpp(120): Expression 6 == a became 6 == Approx( 5.0 )
      </failure>
      <failure message="6 &lt;  Approx( 5.0 )" type="CHECK">
        coverage_maxout.cpp(123): Expression 6 &lt; a became 6 &lt;  Approx( 5.0 )
      </failure>
      <failure message="6 &lt;= Approx( 5.0 )" type="CHECK">
        coverage_maxout.cpp(125): Expression 6 &lt;= a became 6 &lt;= Approx( 5.0 )
      </failure>
    </testcase>
    <testcase classname="coverage_maxout.cpp" name="will end from a std::string exception">
      <error message="exception">
        std::string!
      </error>
    </testcase>
    <testcase classname="coverage_maxout.cpp" name="will end from a const char* exception">
      <error message="exception">
        const char*!
      </error>
    </testcase>
    <testcase classname="coverage_maxout.cpp" name="will end from an unknown exception">
      <error message="exception">
        unknown exception
      </error>
    </testcase>
  </testsuite>
</testsuites>
Program code.
